package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "mimic-fn",
-	"version": "1.2.0",
+	"version": "2.0.0",
```

Comment:
You should never bump the version in other people's repos. It's up to the maintainers to choose when and how to bump the version.

Reply:
According to [username - PR#20](https://github.com/sindresorhus/username/pull/20), I thought making the break change to Node.js 6 would save your time.

Sorry for that 😑

Reply:
No worries :)